_format/1.0/index.md

```diff
@@ -1560,7 +1560,7 @@ is an empty array or an array of [resource identifier objects][resource identifi
 If a client makes a `PATCH` request to a URL from a to-many
 [relationship link][relationships], the server **MUST** either completely
 replace every member of the relationship, return an appropriate error response
-if some resources can not be found or accessed, or return a `403 Forbidden`
+if some resources cannot be found or accessed, or return a `403 Forbidden`
```

Comment:
After reviewing a few sources, it appears that `can not` is just as acceptable as `cannot`. Is there a reason you made this change?

Reply:
Agreed, my mistake. I have reverted this.